test/e2e/replica_set_scale_down/replica_set_scale_down_test.go

```diff
@@ -56,6 +56,7 @@ func TestReplicaSetScaleDown(t *testing.T) {
 			mdbv1.MongoDBCommunityStatus{
 				MongoURI:                   mdb.MongoURI(),
 				Phase:                      mdbv1.Running,
+				Version:                    mdb.Spec.Version,
```

Comment:
ditto

Reply:
good catch, I'll check again to find all other occurrences